arviz/plots/violinplot.py

```diff
@@ -61,6 +62,10 @@ def plot_violin(
         intervals. Defaults to ``True``.
     rug: bool
         If ``True`` adds a jittered rugplot. Defaults to ``False``.
+    side: string
```

Comment:
```suggestion
    side : str, default "both"
```
then the default can also be removed from the description.

(the rest of the docstring is bad example to follow :sweat_smile:, there is no need to fix all of it only to add this one right from the start)